<h2>Description</h2>
<p>These tracks show curated SARS-CoV-2 protein-coding genes conserved within the Sarbecovirus subgenus as determined using PhyloCSF [1], FRESCo [2], and other comparative genomics methods, consistent with experimental evidence in SARS-CoV-2. For a complete description of the evidence, see [3].</p>
<ul>
  <li>
      The <b>PhyloCSF Genes</b> track shows the conserved protein-coding genes, namely ORF1a, ORF1ab, S, ORF3a, ORF3c (a.k.a. ORF3h, ORF3a*, and 3a.iORF1), E, M, ORF6, ORF7a, ORF7b, ORF8, N, and ORF9b (a.k.a. ORF9a).
  </li>
  <br>
  <li>
      The <b>PhyloCSF Rejected Genes</b> track shows other genes that have been proposed that do not show the signature of conserved protein-coding genes or persuasive experimental evidence, and are thus unlikely to be actual protein-coding genes, namely ORF3d, ORF3b, ORF14 (a.k.a. ORF9b, ORF9c), and ORF10. (Two different ORFs have been referred to as ORF3b. We refer to the one with coordinates 25524-25697 as ORF3d. The other, with coordinates 25814-26281, is the ortholog of SARS-CoV-1 ORF3b, but is not an ORF in SARS-CoV-2 due to in-frame stop codons.)
  </li>
</ul>

<h2>Data Access</h2>
<p>
The raw data can be explored interactively with the
<a href="hgTables">Table Browser</a> or combined with other datasets in the
<a href="hgIntegrator">Data Integrator</a> tool.
For automated analysis, the genome annotation is stored in
a bigBed file that can be downloaded from
<a href="http://hgdownload.soe.ucsc.edu/gbdb/wuhCor1/bbi/contacts.bb">the download server</a>.</p>
<p>
Annotations can
be converted from binary to ASCII text by our command-line tool bigBedToBed.
Instructions for downloading this command can be found on our
<a href="http://hgdownload.soe.ucsc.edu/downloads.html#utilities_downloads">utilities page</a>.
The tool can also be used to obtain features within a given range without downloading the file,
for example:</p>
<code>bigBedToBed http://hgdownload.soe.ucsc.edu/gbdb/wuhCor1/bbi/phyloGenes/PhyloCSFgenes.bb -chrom=NC_045512v2 -start=0 -end=29902 stdout</code>
<p>
Please refer to our
<a href="https://groups.google.com/a/soe.ucsc.edu/forum/#!forum/genome">mailing list archives</a>
for questions, or our
<a href="../../FAQ/FAQdownloads.html#download36">Data Access FAQ</a>
for more information.</p>


<h2>Methods</h2>
<p>See [3].</p>

<h2>Credits</h2>
<p>Questions should be directed to <a href="mailto:iljungr@csail.mit.edu"><span>Irwin Jungreis</span></a>.</p>
<p>If you use the SARS-CoV-2 PhyloCSF Genes Track Hub, please cite Jungreis et al. 2020 [3].</p>
<h2>References</h2>

<p><span>[1] </span>Lin MF, Jungreis I, and Kellis M (2011). PhyloCSF: a comparative genomics method to distinguish protein-coding and non-coding regions. <a href="http://bioinformatics.oxfordjournals.org/content/27/13/i275.abstract"><span><i>Bioinformatics</i></span><span> 27:i275-i282</span></a> (ISMB/ECCB 2011).</p>

<p><span>[2] </span>Sealfon RS, Lin MF, Jungreis I, Wolf MY, Kellis M, Sabeti PC (2015). FRESCo: finding regions of excess synonymous constraint in diverse viruses. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4376164/"><span><i>Genome Biol. doi: 10.1186/s13059-015-0603-7</i></span></a>.</p>

<p><span>[3] </span>Jungreis I, Sealfon R, Kellis M (2020). Sarbecovirus comparative genomics elucidates gene content of SARS-CoV-2 and functional impact of COVID-19 pandemic mutations. <a href="https://www.biorxiv.org/content/10.1101/2020.06.02.130955v1"><span><i>bioRxiv doi:10.1101/2020.06.02.130955</i></span></a>.</p>
